### PR TITLE
Add translation interface available in a whole application.

### PR DESCRIPTION
**Pull request by KanarekLife**

### CODE REVIEW

**Comment by KanarekLife** [comment]
I didn't know how to write tests so they're not yet implemented.

File: src/eru.Infrastructure/Translation/DependencyInjection.cs [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            {
+                options.ResourcesPath = "Resources";
+            });
+            services.Configure<RequestLocalizationOptions>(options =>
+            {
+                var cultures = new[]
+                {
+                    new CultureInfo("en"),
+                    new CultureInfo("pl"),
+                };
```
Comment (Bazik33) [inline src/eru.Infrastructure/Translation/DependencyInjection.cs:25]: There should be a way to enumerate cultures from the app.

File: src/eru.Infrastructure/Translation/DependencyInjection.cs [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+                options.DefaultRequestCulture = new RequestCulture("en");
```
Comment (Bazik33) [inline src/eru.Infrastructure/Translation/DependencyInjection.cs:26]:
```suggestion
                options.DefaultRequestCulture = _configuration["DefaultLanguage"]; 
```

DefaultRequestCulture should be taken from app configuration.

File: src/eru.Infrastructure/eru.Infrastructure.csproj [hunk 3]
```diff
@@ -20,4 +20,46 @@
...
+        <DesignTime>True</DesignTime>
+        <AutoGen>True</AutoGen>
+        <DependentUpon>ApplicationDbContext.pl.resx</DependentUpon>
+      </Compile>
+      <Compile Update="Resources\Persistence\ApplicationDbContext.en.Designer.cs">
+        <DesignTime>True</DesignTime>
+        <AutoGen>True</AutoGen>
+        <DependentUpon>ApplicationDbContext.en.resx</DependentUpon>
+      </Compile>
+    </ItemGroup>
```
Comment (Bazik33) [inline src/eru.Infrastructure/eru.Infrastructure.csproj:63]: We don't even have these files in the project.

**Review by Bazik33 — CHANGES_REQUESTED** [review]

File: src/eru.Infrastructure/Translation/DependencyInjection.cs (hunk 1 above)
Comment (KanarekLife) [reply src/eru.Infrastructure/Translation/DependencyInjection.cs:25]: moved to #64

File: src/eru.Infrastructure/Translation/DependencyInjection.cs (hunk 2 above)
Comment (KanarekLife) [reply src/eru.Infrastructure/Translation/DependencyInjection.cs:26]: moved to #64

File: src/eru.Infrastructure/eru.Infrastructure.csproj (hunk 3 above)
Comment (KanarekLife) [reply src/eru.Infrastructure/eru.Infrastructure.csproj:63]: Created by Rider and left by mistake ;)
Create a suggestion.

**Comment by KanarekLife** [comment]
fixes #64

**Review by Bazik33 — APPROVED** [review]
Looking good.

**Pull request merged**